Extension/i18n/rus/package.i18n.json

```diff
@@ -176,7 +176,7 @@
 	"c_cpp.configuration.filesExcludeWhen.description": "Дополнительная проверка элементов того же уровня соответствующего файла. Используйте $(basename) в качестве переменной для соответствующего имени файла.",
 	"c_cpp.configuration.debugger.useBacktickCommandSubstitution.description": "Если задано значение true, для подстановки команд оболочки отладчика будет использоваться устаревший обратный апостроф (`).",
 	"c_cpp.contributes.views.cppReferencesView.title": "C/C++: результаты по другим ссылкам",
-	"c_cpp.contributes.viewsWelcome.contents": "To learn more about launch.json, see [Configuring C/C++ debugging](https://code.visualstudio.com/docs/cpp/launch-json-reference).",
+	"c_cpp.contributes.viewsWelcome.contents": "Дополнительные сведения о launch.json см. в статье [Настройка отладки C/C++] (https://code.visualstudio.com/docs/cpp/launch-json-reference).",
```

Comment:
There is an extra space here that breaks the markdown.  (We had even included a note that this string contains markdown...)